### ISSUE TITLE
publish new activities to the user's followers

### ISSUE THREAD

**Issue by jessehattabaugh** [jessehattabaugh/RetroToot#9]
Continuation of #4 

Currently when the `GetRecentAchievements` task runs it updates the `User.recentAchievements` only. 

It should compare the current `recentAchievements` to the existing `recentAchievements` and queue another task for each new one.  This new task can publish the activity to the user's followers once #3 is complete.
